admin_guide/sysctls.adoc

```diff
@@ -133,6 +133,25 @@ kubeletArguments:
     - "kernel.msg*,net.ipv4.route.min_pmtu"
 ----
 
+. Add the required sysctl parameter to the `allowedUnsafeSysctls` list in a security context constraint (SCC) object. It is recommened to use a new SCC object:
++
+[source,yaml]
+----
+[...]
+allowedUnsafeSysctls:
```

Comment:
line 141 and 142 have format issue, should be:
allowedUnsafeSysctls:
  - kernel.msg*
  - net.ipv4.route.min_pmtu